photon-client/src/components/cameras/CameraCalibrationInfoCard.vue

```diff
@@ -97,197 +95,221 @@ const importCalibration = async () => {
-                    ? isNaN(videoFormat.mean)
-                      ? "NaN"
-                      : videoFormat.mean.toFixed(2) + "px"
-                    : "-"
-                }}
-              </td>
-            </tr>
-            <tr>
-              <td>Horizontal FOV</td>
-              <td>{{ videoFormat.horizontalFOV !== undefined ? videoFormat.horizontalFOV.toFixed(2) + "°" : "-" }}</td>
-            </tr>
-            <tr>
-              <td>Vertical FOV</td>
-              <td>{{ videoFormat.verticalFOV !== undefined ? videoFormat.verticalFOV.toFixed(2) + "°" : "-" }}</td>
-            </tr>
-            <tr>
-              <td>Diagonal FOV</td>
-              <td>{{ videoFormat.diagonalFOV !== undefined ? videoFormat.diagonalFOV.toFixed(2) + "°" : "-" }}</td>
-            </tr>
-            <!-- Board warp, only shown for mrcal-calibrated cameras -->
-            <tr
-              v-if="
-                useCameraSettingsStore().getCalibrationCoeffs(props.videoFormat.resolution)?.calobjectWarp?.length === 2
-              "
-            >
-              <td>Board warp, X/Y</td>
-              <td>
-                {{
-                  useCameraSettingsStore()
-                    .getCalibrationCoeffs(props.videoFormat.resolution)
-                    ?.calobjectWarp?.map((it) => (it * 1000).toFixed(2) + " mm")
-                    .join(" / ")
-                }}
-              </td>
-            </tr>
-          </tbody>
-        </template>
-      </v-simple-table>
-      <hr style="width: 100%" class="ma-6" />
-      <v-card-subtitle>Per Observation Details</v-card-subtitle>
-      <v-data-table
-        dense
-        style="width: 100%"
-        class="pl-2 pr-2"
-        :headers="[
-          { text: 'Observation Id', value: 'index' },
-          { text: 'Mean Reprojection Error', value: 'mean' }
-        ]"
-        :items="getObservationDetails()"
-        item-key="index"
-        show-expand
-        expand-icon="mdi-eye"
-      >
-        <template #expanded-item="{ headers, item }">
-          <td :colspan="headers.length">
-            <div style="display: flex; justify-content: center; width: 100%">
-              <img :src="item.snapshotSrc" alt="observation image" class="snapshot-preview pt-2 pb-2" />
-            </div>
-          </td>
-        </template>
-      </v-data-table>
-    </v-row>
-    <v-row v-else class="pt-2 mb-0 pb-0">
-      The selected video format doesn't have any additional information as it has yet to be calibrated.
-    </v-row>
-  </v-card>
+          <template #expanded-item="{ headers, item }">
+            <td :colspan="headers.length">
+              <div style="display: flex; justify-content: center; width: 100%">
+                <img
+                  :src="`http://${address}/api/utils/getCalSnapshot?width=${props.videoFormat.resolution.width}&height=${props.videoFormat.resolution.height}&snapshotIdx=${item.index}&cameraIdx=${useStateStore().currentCameraIndex}`"
```

Comment:
Ditto on URLSearchParams. Injection stuff shouldn't be here either, make it a computed property and then both the axios request and this can pull from it.